pkg/apis/cluster/v1alpha1/types.go

```diff
@@ -150,6 +158,24 @@ type NodeSummary struct {
 	Used corev1.ResourceList `json:"used,omitempty"`
 }
 
+// Nodes represents the numbers of all nodes and ready nodes in a specific cluster.
+type Nodes struct {
+	// TotalNum is the total number of nodes in the cluster.
+	TotalNum int `json:"totalNum,omitempty"`
+	// ReadyNum is the number of ready nodes in the cluster.
+	ReadyNum int `json:"readyNum,omitempty"`
+}
+
+// Resources represents the allocatable, allocating and allocated resources in a specific cluster.
+type Resources struct {
+	// Allocatable represents the allocatable resources across all nodes.
+	Allocatable corev1.ResourceList `json:"allocatable,omitempty"`
+	// Allocating represents the allocating resources across all nodes whose node name has not been set.
```

Comment:
This sentence might something wrong because we can't set nodes to a `resource` or `node`.